corehq/apps/cloudcare/static/cloudcare/js/form_entry/utils.js

```diff
@@ -73,7 +73,7 @@ hqDefine("cloudcare/js/form_entry/utils", function () {
      * @param {Object} clearCallBack - function to call back after clearing the input
      * @param {Object} initialPageData - initial_page_data object
      */
-    module.renderMapboxInput = function (divId, itemCallback, clearCallBack, initialPageData) {
+    module.renderMapboxInput = function (divId, itemCallback, clearCallBack, initialPageData, inputOnKeyDown) {
```

Comment:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js#L140 what about this declaration for the adjusted params?

Reply:
Thanks for pointing this out. I have added a check for inputOnKeyDown on `renderMapboxInput` as the code that you pointed out does not have a `inputOnKeyDown` or similar function present. https://github.com/dimagi/commcare-hq/pull/32637/commits/ecb74b75dc35a8a3397d3eb1e1c338833ad3ea7b